src/cmds/docs/index.ts

```diff
@@ -67,7 +76,17 @@ export default class DocsCommand extends Command {
 
     Command.debug(`number of files: ${files.length}`);
 
-    if (!files.length) {
+    const changes: string[] = [];
+    if (deleteMissing) {
+      const docs = await getDocs(key, selectedVersion);
+      const docSlugs = docs.map(({ slug }: { slug: string }) => slug);
+      const fileSlugs = new Set(files.map(getSlug));
+      const slugsToDelete = docSlugs.filter((slug: string) => !fileSlugs.has(slug));
+      const deletedDocs = await Promise.all(
+        slugsToDelete.map((slug: string) => deleteDoc(key, selectedVersion, dryRun, slug, this.cmdCategory))
+      );
+      changes.push(...deletedDocs);
+    } else if (!files.length) {
```

Comment:
What happens if the `files` array is empty and someone passes in the `--deleteMissing` flag? Could you move this logic below this block below this block? https://github.com/readmeio/rdme/blob/0292a09eabc9e0beb0e982f6f24bed4c2f757a41/src/cmds/docs/index.ts#L89-L91

Reply:
Actually that's on purpose. If all files were deleted from the folder then technically all files should be removed from ReadMe. But I do see how it may be a problem if someone accidentally passes a wrong empty folder.
Maybe in this case I can add a confirmation prompt? "This action will remove all files from ReadMe, are you sure you with to continue?" WDYT?

Reply:
Added the confirmation prompt for now, LMK what you think